<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>WindowMain</class>
	<widget class="QMainWindow" name="WindowMain">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>290</width>
				<height>252</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Pivot</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../icons/ToolSeq.svg</normaloff>../icons/ToolSeq.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Widget_00">
			<layout class="QVBoxLayout" name="verticalLayout">
				<property name="spacing">
					<number>2</number>
				</property>
				<property name="leftMargin">
					<number>2</number>
				</property>
				<property name="topMargin">
					<number>2</number>
				</property>
				<property name="rightMargin">
					<number>2</number>
				</property>
				<property name="bottomMargin">
					<number>2</number>
				</property>
				<item>
					<widget class="QWidget" name="Widget_01" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout">
							<property name="spacing">
								<number>10</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="label_6">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Transforms</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_5">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>:</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="Label_TransformsCount">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>0</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Widget_011" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout_6">
										<property name="spacing">
											<number>5</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QCheckBox" name="Check_X">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="layoutDirection">
													<enum>Qt::RightToLeft</enum>
												</property>
												<property name="text">
													<string>X</string>
												</property>
												<property name="checked">
													<bool>true</bool>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QCheckBox" name="Check_Y">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="layoutDirection">
													<enum>Qt::RightToLeft</enum>
												</property>
												<property name="text">
													<string>Y</string>
												</property>
												<property name="checked">
													<bool>true</bool>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QCheckBox" name="Check_Z">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="layoutDirection">
													<enum>Qt::RightToLeft</enum>
												</property>
												<property name="text">
													<string>Z</string>
												</property>
												<property name="checked">
													<bool>true</bool>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_02" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_4">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPlainTextEdit" name="Text_Transforms">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="lineWrapMode">
										<enum>QPlainTextEdit::NoWrap</enum>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="Line" name="Widget_03">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>0</width>
								<height>10</height>
							</size>
						</property>
						<property name="frameShadow">
							<enum>QFrame::Plain</enum>
						</property>
						<property name="lineWidth">
							<number>2</number>
						</property>
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_04" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_5">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QCheckBox" name="Check_MinMaxCenRotPiv">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>RotPiv</string>
									</property>
									<property name="checked">
										<bool>true</bool>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QCheckBox" name="Check_MinMaxCenScaPiv">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>ScaPiv</string>
									</property>
									<property name="checked">
										<bool>true</bool>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Minimum">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Min</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Maximum">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Max</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Center">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Cen</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="Line" name="Widget_05">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>0</width>
								<height>10</height>
							</size>
						</property>
						<property name="frameShadow">
							<enum>QFrame::Plain</enum>
						</property>
						<property name="lineWidth">
							<number>2</number>
						</property>
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_06" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,3">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_CreateLocators">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Create Locators from Transforms Piv</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QComboBox" name="Combo_CreateLocatorsType">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="currentIndex">
										<number>0</number>
									</property>
									<item>
										<property name="text">
											<string>RotPiv</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>ScaPiv</string>
										</property>
									</item>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="Line" name="Widget_07">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>0</width>
								<height>10</height>
							</size>
						</property>
						<property name="frameShadow">
							<enum>QFrame::Plain</enum>
						</property>
						<property name="lineWidth">
							<number>2</number>
						</property>
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_08" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_Locator" native="true">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<layout class="QHBoxLayout" name="horizontalLayout_2">
										<property name="spacing">
											<number>5</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="label">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Locator</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_7">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="Line_Locator">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_09" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1,1">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_MoveTransformsPivot">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Move Transforms Pivot to Loc</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QCheckBox" name="Check_MoveTransformsPivotRotPiv">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>RotPiv</string>
									</property>
									<property name="checked">
										<bool>true</bool>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QCheckBox" name="Check_MoveTransformsPivotScaPiv">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>ScaPiv</string>
									</property>
									<property name="checked">
										<bool>true</bool>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_10" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_9" stretch="8,3">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_MatchTransformsPivot">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Match Transforms Pivot to Loc</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QComboBox" name="Combo_MatchTransformsPivotType">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="currentIndex">
										<number>0</number>
									</property>
									<item>
										<property name="text">
											<string>RotPiv</string>
										</property>
									</item>
									<item>
										<property name="text">
											<string>ScaPiv</string>
										</property>
									</item>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>